content/2022/prepoznavanje-govora.md

```diff
@@ -2,3 +2,237 @@
 title: Prepoznavanje govora
 summary: Projekat iz prepoznavanja govora rađen na letnjem kampu za stare polaznike 2022. godine od Dimitrija Pešića i Lazara Zubovića.
 ---
+### Apstrakt
+
+### Apstrakt na engleskom
+### Uvod
+Projekat "Prepoznavanje govora" pomaže pri rešavanju popularne dileme u AI tehnologiji, a to je kako da se glas pretvori u kucani tekst. Prepoznavanje govora je proces osposobljavanja nekog modela da prepozna i odreaguje na zvuk proizveden ljudskim govorom. Model uzima audio signal u formi talasa, izvlači iz njega podatke, obrađuje ih, prepoznaje i preduzima određene korake u zavisnosti od rezultata.
```

Comment:
```suggestion
Projekat "Prepoznavanje govora" pomaže pri rešavanju popularne dileme u AI tehnologiji, a to je kako da se glas pretvori u kucani tekst. Prepoznavanje govora je proces osposobljavanja nekog modela da prepozna i odreaguje na zvuk proizveden ljudskim govorom. Model uzima audio signal u formi talasa, izvlači iz njega podatke, obrađuje ih i identifikuje izgovorenu reč.
```

Reply:
Nije od krucijalnog značaja ali bilo bi lepo da na par mesta upotrebite neku reč koja je opisnija od one koja je nastala od reči "prepoznati". S jedne strane zato što u kontekstu u kom opisujete, baš je na nekim mestima generičko i ne daje puno informacija. S druge strane, u prvih par pasusa previše ponavljate ovu reč.